scripts/mea/et/internal_witness.py

```diff
@@ -274,6 +274,29 @@ def process_verifier_notes(self):
                 env = self._env_models_json[file_id][start_line]
                 comment = env["comment"]
                 relevant = env.get("relevant", False)
+                name = env.get("name", None)
+                action_type = env.get("type", None)
+                self._logger.debug(f"type: {action_type} for name {name}")
+                action_type_data = {}
+                if name is not None:
+                    action_type_data[name] = []
+                if action_type == "ACTION_BEGIN":
+                    action_begin_dict = {}
+                    action_begin_dict['env'] = comment
+                    action_begin_dict['file'] = file_id
+                    action_begin_dict['source'] = edge.get("source", None)
+                    action_begin_dict['enter'] = self.add_function(name)
+                    action_begin_dict['start line'] = start_line
```

Comment:
I'm not sure, that is all tags here. At least `threadId` may also present. I suggest to iterate over all tags of template edge